packages/block-editor/src/store/selectors.js

```diff
@@ -28,8 +28,8 @@ import {
 	getPossibleBlockTransformations,
 	parse,
 } from '@wordpress/blocks';
-import { SVG, Rect, G, Path } from '@wordpress/components';
 import { Platform } from '@wordpress/element';
+import { reusableBlockType } from '@wordpress/icons';
```

Comment:
I think maybe we should call this one "symbol" and "symbol-filled" at the icon level